Source/Scene/Cesium3DTilesetTraversal.js

```diff
@@ -483,13 +491,13 @@ define([
                     // Always load tiles in the base traversal
                     // Select tiles that can't refine further
                     loadTile(tileset, tile, frameState);
-                    if (!refines && parentRefines) {
+                    if (stoppedRefining) {
                         selectDesiredTile(tileset, tile, frameState);
                     }
                 } else {
                     // Load tiles that are not skipped or can't refine further. In practice roughly half the tiles stay unloaded.
                     // Select tiles that can't refine further. If the tile doesn't have loaded content it will try to select an ancestor with loaded content instead.
-                    if (!refines) { // eslint-disable-line
+                    if (stoppedRefining) { // eslint-disable-line
```

Comment:
You can get rid of the comment disabling eslint if you bring this if statement up a block. I don't think it hurts readability at all.

Reply:
I know this is all minor but I prefer this organization because it's easier to see the how the base traversal and skip traversal operate when they are separated like this.

Reply:
We have eslint rules for a reason though, adding disable-line because you don't like a rule is kind of a bad precedent.  If code needs clarifying, I would indicate that with a comment, not some layout that only makes sense to people who already know the code.